docs/guides/Quick Start/02-product-management.md

```diff
@@ -0,0 +1,547 @@
+# Product Overview
+
+This section explains the handling of the product data structure.
+
+To start easy, below is a sample request to create a product with simple data from its complex API schema.
+
+## Create a new product with a simple payload
+
+A product has only a handful of required fields:
+
+* name [string]
+* productNumber [string]
+* taxId [string]
+* price [object]
+* stock [int]
+
+The smallest required payload for a product can therefore be as follows:
+
+```json http
+{
+  "method": "post",
+  "url": "http://localhost/api/product",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  },
+  "body": {
+    "name": "test",
+    "productNumber": "random",
+    "stock": 10,
+    "taxId": "a5da76b447db4d0aba62e6512dadf45b",
+    "price": [
+        {
+            "currencyId" : "b7d2554b0ce847cd82f3ac9bd1c0dfca", 
+            "gross": 15, 
+            "net": 10, 
+            "linked" : false
+        }
+    ]
+      }
+    }
+  }
+```
+
+```json json_schema
+{
+  "type": "object",
```

Comment:
You can add a title to make it more readable

```suggestion
  "type": "object",
  "title": "MinimalProductPayload",
```

![image](https://user-images.githubusercontent.com/7780750/203088055-7fb94f1b-0c2c-4eea-aa28-1c623a35b87f.png)